### PR TITLE
fix: omit private fields in wallet api

### LINKED ISSUE #145: Hide inappropriate fields in GET /wallets
```
{
    "wallets": [
        {
            "id": 27,
            "type": "p",
            "name": "Conrad.Hills",
            "password": "db6331ea3828ff53fd2061ee64d712312d1d6462d0d214716abbc1ccc3aff85a46fc1acb655c94e4c9da34a91283c31bbb7ccf243f1ea0e456a5a05e96b5f07f",
            "salt": "jqQC8QX_i2zX80Q",
            "logo_url": null,
            "email": null,
            "phone": null,
            "tokens_in_wallet": 10
        }
    ]
}
```

Hide fields: `password`, `salt`, `type`

### PR DESCRIPTION
**Pull request by zo-chu**
fixes #145 #178

### CODE REVIEW

File: server/routes/walletRouter.spec.js
Comment (dadiorchen) [inline server/routes/walletRouter.spec.js:87]: This is an interesting case, I thought.
Do you think we can just test the omit function in util (against the fn directly), for covering the code? In that way, it would be more easy?
I am considering using util/model pieces, and just write unit test for them, to bring some relief for writing tests.

Comment (dadiorchen) [reply server/routes/walletRouter.spec.js:87]: @deepwinter is this your opinion you ever mentioned?

**Review by dadiorchen — APPROVED** [review]

File: server/routes/walletRouter.spec.js
Comment (zo-chu) [reply server/routes/walletRouter.spec.js:87]: @dadiorchen I think we could use a ready made omit function like with a lodash library and then this test would stay the same. In my opinion, it's more important to have a test that checks that the fields are omitted (aka behaviour test) than test implementation - omit function. Tell me what you think.
PS I've noticed that there are minimal amount of dependencies in the project, how do you feel about importing new libraries? like lodash?

Comment (deepwinter) [reply server/routes/walletRouter.spec.js:87]: I definitely support he use of lodash in all our node.js services.

Comment (dadiorchen) [reply server/routes/walletRouter.spec.js:87]: I didn't use lodash, but I'd like to try it, so feel free to add it.
About writing the test, these days I'm thinking of the strategy, my thoughts on this is that we use unit tests to cover edge cases, and use integration test to cover major workflow, because the latter is more heavy bringing burden for devs.

**Review by dadiorchen — APPROVED** [review]

File: server/routes/walletRouter.spec.js
Comment (dadiorchen) [reply server/routes/walletRouter.spec.js:87]: Oh, this isn't integration, but my point is that use the most easy way to cover code.

Comment (zo-chu) [reply server/routes/walletRouter.spec.js:87]: cool, I've added lodash and now it's nicer. It's super easy to use and very handy

Comment (zo-chu) [reply server/routes/walletRouter.spec.js:87]: I can't merge this though (no write access), so unless any changes required, please feel free to merge @dadiorchen or @deepwinter

**Pull request merged**
